packages/lesswrong/themes/userThemes/darkMode.ts

```diff
@@ -271,7 +271,7 @@ export const darkModeTheme: UserThemeSpecification = {
       sidenoteBackground: "rgba(180,160,160,.1)",
     },
     border: {
-      itemSeparatorBottom: 'none',
+      itemSeparatorBottom: shadePalette.greyBorder("1px", .2),
```

Comment:
Won't this now add a double bottom border, given the other bottom-border change for the PostsItems always applies.